index.html

```diff
@@ -1,68 +1,80 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" href="public/image/star-1.svg">
-    <link rel="stylesheet" href="/style.css">
-    <link rel="stylesheet" href="/ui-kit.css">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Ink. House</title>
-  </head>
-  <body>
-    <header class="header container">
-      <a class="header-logo" href="/">
-        Ink. House
-      </a>
-      <nav class="header-menu">
-        <label class="label-header-button header-burger-button green-button border-none" for="header-button">
-          <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
-          <path fill-rule="evenodd" clip-rule="evenodd" d="M4.6875 21.5625C4.6875 21.3139 4.78627 21.0754 4.96209 20.8996C5.1379 20.7238 5.37636 20.625 5.625 20.625H24.375C24.6236 20.625 24.8621 20.7238 25.0379 20.8996C25.2137 21.0754 25.3125 21.3139 25.3125 21.5625C25.3125 21.8111 25.2137 22.0496 25.0379 22.2254C24.8621 22.4012 24.6236 22.5 24.375 22.5H5.625C5.37636 22.5 5.1379 22.4012 4.96209 22.2254C4.78627 22.0496 4.6875 21.8111 4.6875 21.5625ZM4.6875 14.0625C4.6875 13.8139 4.78627 13.5754 4.96209 13.3996C5.1379 13.2238 5.37636 13.125 5.625 13.125H24.375C24.6236 13.125 24.8621 13.2238 25.0379 13.3996C25.2137 13.5754 25.3125 13.8139 25.3125 14.0625C25.3125 14.3111 25.2137 14.5496 25.0379 14.7254C24.8621 14.9012 24.6236 15 24.375 15H5.625C5.37636 15 5.1379 14.9012 4.96209 14.7254C4.78627 14.5496 4.6875 14.3111 4.6875 14.0625ZM4.6875 6.5625C4.6875 6.31386 4.78627 6.0754 4.96209 5.89959C5.1379 5.72377 5.37636 5.625 5.625 5.625H24.375C24.6236 5.625 24.8621 5.72377 25.0379 5.89959C25.2137 6.0754 25.3125 6.31386 25.3125 6.5625C25.3125 6.81114 25.2137 7.0496 25.0379 7.22541C24.8621 7.40123 24.6236 7.5 24.375 7.5H5.625C5.37636 7.5 5.1379 7.40123 4.96209 7.22541C4.78627 7.0496 4.6875 6.81114 4.6875 6.5625Z" fill="#4C5866"/>
-          </svg>
-        </label>
-        <input type="checkbox" class="header-checkbox" id="header-button">
-          </input>
-          
 
-          <div class="nav-list"> 
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" href="public/image/star-1.svg">
+  <link rel="stylesheet" href="/style.css">
+  <link rel="stylesheet" href="/ui-kit.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Ink. House</title>
+</head>
 
-            <ul class="header-menu-list">
-      
+<body>
+  <header class="header container">
```

Comment:
из-за того, что на хэдере навешен класс контейнера, его ширина ограничена и на больших экранах не растягивается на всю ширину. Нужно хэдеру указать ширину 100%, создать внутри div с классом контейнера и задать стили, чтобы выровнять всё.

```suggestion
  <header class="header">
    <div class="container">  
```

<img width="1725" alt="Screenshot 2024-06-02 at 00 15 16" src="https://github.com/richardfosfor/ink.house/assets/39421667/7a1d7acc-8629-449a-ae9e-22c6b37bfc4a">